docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Command line options](api.md#command-line-interface).
+- [Device Model](device_model.md).
+- Device tree, device list, ACPI, Hyper-V enlightenments and any other features
+  exposed to guest.
+- KVM compatibility.
+- UEFI compatibility.
+- Rust edition compatibility.
+
+This list might be by far incomplete. Still this document strives to gather any
+possible indicators to be desperately considered relevant for the stability context.
+
+## Experimental features
+
+Features that are being actively developed and are possibly incomplete or known to be
+unstable are considered experimental. These features are in no case recommended for
+use in production.
+
+Features currently considered experimental:
+
+- TDX.
+- vfio-user.
+- vDPA.
+
+## Security
+
+Security issues usually involve only defects directly related to Cloud Hypervisor.
+
+While security issues in the compiler or third party components are out of scope as
+they can't be influenced by the Cloud Hypervisor project, still fixes to affected
+code or changes to ensure a non vulnerable dependency version is used might trigger
+a new point release. However, these measures won't affect possibly already released
+tarballs or binary artifacts.
```

Comment:
Not sure what this sentence is trying to say. Surely old releases are vulnerable if they embed a bad dependency.

Reply:
It says as simple as it sounds - none of released artifacts or tags will be replaced after a release is announced. Instead, there would be a next release fixing the issue. One can wonder and while for many this would be the expected strategy, it was not once i could hear folks would expect things could to be replaced even after some time. Thus, explicitly telling that.

thanks

Reply:
@weltling I've never heard of an Open Source project retrospectively changing the content of a tag or a released tarball. There is no need to say it.

Reply:
I agree. The only thing we should say here is something along those lines: "Security fixes should be included in a new point release"

Reply:
I only mentioned "users might be asking", so kinda eliminating questions beforehand, projects wouldn't do it OFC :)

Ok, so i removed this para and put the suggestion from Sebastien instead.

For the CVE part also - yep, so if GH has a mechanism, it would be it at least. Here is probably the only important that one is able to communicate, if a release is security relevant. CVE might be still requested afterwards, just in case.

Thanks